suite-native/module-settings/src/screens/SettingsViewOnly.tsx

```diff
@@ -1,33 +1,117 @@
 import { useDispatch, useSelector } from 'react-redux';
+import { useState } from 'react';
+
+import { A } from '@mobily/ts-belt';
 
 import { Screen, ScreenSubHeader } from '@suite-native/navigation';
 import { Box, Button, Card, HStack, IconButton, Text } from '@suite-native/atoms';
 import { Translation, useTranslate } from '@suite-native/intl';
 import { selectPhysicalDevices, toggleRememberDevice } from '@suite-common/wallet-core';
 import { TrezorDevice } from '@suite-common/suite-types';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useAlert } from '@suite-native/alerts';
+import { useToast } from '@suite-native/toasts';
 
-export const SettingsViewOnly = () => {
+import { AboutViewOnlyBottomSheet } from '../components/AboutViewOnlyBottomSheet';
+import { SecuredCoinsSvg } from '../assets/SecuredCoinsSvg';
+
+type AboutProps = {
+    onPressAbout: () => void;
+};
+
+const About = ({ onPressAbout }: AboutProps) => (
+    <Box marginVertical="medium" marginHorizontal="extraLarge" alignItems="center">
+        <Text variant="hint" textAlign="center" color="textSubdued">
+            <Translation
+                id="moduleSettings.viewOnly.subtitle"
+                values={{
+                    about: chunk =>
+                        chunk && (
```

Comment:
```suggestion

```

Why is the check for chunk necessary? It should be always defined if the function gets to the execution.  Or am I missing something?

Reply:
59fad99c60